### ISSUE TITLE
ualterms namespace uri error

### ISSUE THREAD

**Issue by anayram** [ualbertalib/HydraNorth#986]
@piyapongch found a namespace uri error coming through OAI metadata (ualterms vocabulary):
`http://terms.library.library.ca/identifiers/`

I checked migration data but the problem seems to come from the ruby rdf vocabulary file: [HydraNorth/lib/rdf_vocabularies/ualterms.rb](https://github.com/ualbertalib/HydraNorth/blob/6d09ff5a4523dd528ff389bda469963cc8efff1f/lib/rdf_vocabularies/ualterms.rb)

In case this triggers other changes I am adding a ticket.

**Comment by piyapongch** [comment]
Here are registered namespaces in Fedora 4.

PREFIX premis: <http://www.loc.gov/premis/rdf/v1#>
PREFIX nt: <http://www.jcp.org/jcr/nt/1.0>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX hydragroup: <http://projecthydra.org/ns/auth/group#>
PREFIX hydraacl: <http://projecthydra.org/ns/auth/acl#>
PREFIX xsi: <http://www.w3.org/2001/XMLSchema-instance>
PREFIX mode: <http://www.modeshape.org/1.0>
PREFIX xmlns: <http://www.w3.org/2000/xmlns/>
PREFIX oai: <http://www.openarchives.org/OAI/2.0/>
PREFIX xml: <http://www.w3.org/XML/1998/namespace>
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX model: <info:fedora/fedora-system:def/model#>
PREFIX vivo: <http://vivoweb.org/ontology/core#>
PREFIX fedoraconfig: <http://fedora.info/definitions/v4/config#>
PREFIX mix: <http://www.jcp.org/jcr/mix/1.0>
PREFIX foaf: <http://xmlns.com/foaf/0.1/>
PREFIX relsext: <info:fedora/fedora-system:def/relations-external#>
PREFIX image: <http://www.modeshape.org/images/1.0>
PREFIX sv: <http://www.jcp.org/jcr/sv/1.0>
PREFIX test: <info:fedora/test/>
PREFIX ualterms: <http://terms.library.ualberta.ca/>
PREFIX uatermsid: <http://terms.library.library.ca/identifiers/>
PREFIX scholarsphere: <http://scholarsphere.psu.edu/ns#>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX fedora: <http://fedora.info/definitions/v4/repository#>
PREFIX marcrel: <http://id.loc.gov/vocabulary/relators/>
PREFIX ldp: <http://www.w3.org/ns/ldp#>
PREFIX bibo: <http://purl.org/ontology/bibo/>
PREFIX webacl: <http://www.w3.org/ns/auth/acl#>
PREFIX xs: <http://www.w3.org/2001/XMLSchema>
PREFIX etdms: <http://www.ndltd.org/standards/metadata/etdms/1.0/>
PREFIX dc: <http://purl.org/dc/elements/1.1/>

**Comment by anayram** [comment]
PREFIX uatermsid: http://terms.library.library.ca/identifiers/
needs correcting as well

**Comment by piyapongch** [comment]
I have fixed the uatermsid in OAI provider to match data in Fedora. The metadata fields for uatermsid prefix properties are showing in correct metadata fields in OAI metadata results. 

The OAI provider will need to be changed back to the correct prefix URI when the data has been fixed. Otherwise the metadata fields will not show up in the OAI records.

The fixes has been deployed on Plano. I have also changed the OAI container resource from /oai/setspec to /oai to reflect it's contents.
